src/MediaWiki.ts

```diff
@@ -181,15 +184,12 @@ class MediaWiki {
     const baseUrlDirector = new BaseURLDirector(this.baseUrl.href)
```

Comment:
This whole function/method `initMWApis()` should disappear:
* `baseUrlDirector` should be make class public variable a `this.urlDirector`
* BaseURLDirector should take all the necessary arguments to be able to deliver all the necessary paths:`buildWikimediaDesktopApiUrl`, `buildVisualEditorURL` ...
* `BaseURLDirector` method `buildWikimediaDesktopApiUrl`, `buildVisualEditorURL` ... should be made lazy and renamed in `getWikimediaDekstopApiUrl()`, ....
* Mediawiki method `urlDirector` should be used in place of all these variables.